src/test/javascript/spec/component/import/exercise-import.component.spec.ts

```diff
@@ -11,6 +11,7 @@ import { CodeAnalysisPagingService } from 'app/exercises/programming/manage/serv
 import { ProgrammingExercisePagingService } from 'app/exercises/programming/manage/services/programming-exercise-paging.service';
 import { QuizExercisePagingService } from 'app/exercises/quiz/manage/quiz-exercise-paging.service';
 import { ExerciseImportComponent } from 'app/exercises/shared/import/exercise-import.component';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we need this?

Reply:
I wish I knew.
Removing it, causes eslint to fail claiming `PagingService` is not used as a type. 
This however contradicts [line 265](https://github.com/ls1intum/Artemis/blob/8349c46a8f26b7f3594282aec62bd6580fa4275b/src/test/javascript/spec/component/import/exercise-import.component.spec.ts#L265) and removing the import causes errors.

Reply:
Removing the line works fine for me.